packages/web-components/fast-components-msft/src/tree-item/tree-item.styles.ts

```diff
@@ -86,7 +86,7 @@ export const TreeItemStyles = css`
         ${
             /* Font size should be based off calc(1em + (design-unit + glyph-size-number) * 1px) - 
             update when density story is figured out */ ""
-        } font-size: calc(1em + var(--type-ramp-base-font-size));
+        } font-size: calc(1em + (var(--design-unit) + 16) * 1px);
```

Comment:
Why isn't this on the typeramp?

Reply:
It's not related to the typeramp, it's related to a var we don't have included yet. Imaging the type size changing on the page and the nesting changing. That seems broken to me. It appears as though we were using the type ramp value to get the number 16. Rather, the original intent of 16 was related to horizontal spacing.